### PR TITLE
Update reading-time.mdx

### LINKED ISSUE #10519: Add Reading Time Recipe needs to be update to reflect new render method.
### 📚 Subject area/topic

Recipes

### 📋 Page(s) affected (or suggested, for new content)

https://docs.astro.build/en/recipes/reading-time/

### 📋 Description of content that is out-of-date or incorrect

The line `const { Content, remarkPluginFrontmatter } = await entry.render();` in the code sample for dynamically generated pages needs to be updated to `const { Content, remarkPluginFrontmatter } = await render(entry);` to reflect the new use of render.

`params: { slug: entry.slug },` should be updated to `params: { slug: entry.id },`.

I'm willing to do a PR to correct.

### 🖥️ Reproduction in StackBlitz (if reporting incorrect content or code samples)

https://codesandbox.io/p/devbox/billowing-fast-flypqx?workspaceId=ws_G4yXD7UznkG3Cy1arhHtf1

### PR DESCRIPTION
**Pull request by enigmalea**
#### Description (required)

Updates Reading Time recipe to reflect recent changes to the render method.

#### Related issues & labels (optional)

- Closes #10519 
- Suggested label: 5.0, code snippet update

#### First-time contributor to Astro Docs?

If you are a member of the Astro Discord, please add your username in the description so we can welcome you there!

I'm enigmalea on Discord as well!

### CODE REVIEW

File: src/content/docs/en/recipes/reading-time.mdx [hunk 1]
```diff
@@ -76,18 +76,18 @@ Create a [remark plugin](https://github.com/remarkjs/remark) which adds a readin
 
     ```astro title="src/pages/posts/[slug].astro" "const { Content, remarkPluginFrontmatter } = await entry.render();" "<p>{remarkPluginFrontmatter.minutesRead}</p>"
```
Comment (ArmandPhilippot) [inline src/content/docs/en/recipes/reading-time.mdx:77]: The code snippet uses [Expressive code](https://expressive-code.com/key-features/text-markers/#marking-individual-text-inside-lines) to highlight some parts. So the current text marker no longer matches your edit.
```suggestion
    ```astro title="src/pages/posts/[slug].astro" "const { Content, remarkPluginFrontmatter } = await render(entry);" "<p>{remarkPluginFrontmatter.minutesRead}</p>"
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Thanks for spotting this, and welcome to Team Docs! 🎉 

I left a suggestion to update the highlighting as well.

File: src/content/docs/en/recipes/reading-time.mdx (hunk 1 above)
Comment (enigmalea) [reply src/content/docs/en/recipes/reading-time.mdx]: Expressive Code highlighting has been updated!

**Review by ArmandPhilippot — APPROVED** [review]
Thanks for the update, LGTM now!

**Review by sarah11918 — APPROVED** [review]
Thank you @enigmalea , looks great! We appreciate you taking the time to update this example, and are happy to welcome you to Team Docs! 🥳

**Pull request merged**